src/useCookieConsent.ts

```diff
@@ -14,20 +14,31 @@ import { allCookiesSetToValue, allPropsApproved } from './utils';
 export const useCookieConsent = (
   options?: CookieConsentOptions
 ): CookieConsentHookState => {
+  const storedConsent =
+    typeof options !== 'undefined' && 'storage' in options
```

Comment:
I would extract this whole line into a separate constant called `doesStorageExist` and then just reference it here. Makes it more readable by explaining what this logical expression is doing.